consensus/types/src/execution_payload.rs

```diff
@@ -56,4 +57,15 @@ impl<T: EthSpec> ExecutionPayload<T> {
             transactions: VariableList::empty(),
         }
     }
+
+    #[allow(clippy::integer_arithmetic)]
+    /// Returns the maximum size of an execution payload.
+    pub fn max_execution_payload_size() -> usize {
+        // Fixed part
+        Self::empty().as_ssz_bytes().len()
+        // length of List * size_of(uint8)
+        + (T::max_extra_data_bytes() * <u8 as Encode>::ssz_fixed_len())
```

Comment:
What is this calculating? 
The comment says `length of list` * `size_of(uint8)`. 

It seems the units we are calculating here are bytes. So the multiplication should go like, elements_in_list (dimensionless) * size_in_bytes_of_elements (bytes/element). 

The size of a u8 is always going to be 8, so I think its fine to use that constant.

 So, for the dimensions to work, the left side has to be dimensionless. `max_extra_data_bytes()` seems like it is giving a result in bytes already? If that's the case multiplying by 8 will give size in bits right?

Reply:
`<u8 as Encode>::ssz_fixed_len()` gives the size in bytes = 1. So we are doing `T::max_extra_data_bytes() * 1` here. 
I thought it'd be consistent to not use constants in the calculation here in case we get another variable length value to calculate here later on.

Sorry the comments were bad here. Cleared it up a bit in 7d3aad2 . Let me know if it's okay :)